src/SdssShape.cc

```diff
@@ -597,18 +597,16 @@ bool getAdaptiveMoments(ImageT const &mimage, double bkgd, double xcen, double y
                 if (!(shape->flags[SdssShapeAlgorithm::UNWEIGHTED.number])) {
                     Matrix4d fisher = calc_fisher(*shape, bkgd_var);  // Fisher matrix
                     Matrix4d cov = fisher.inverse();
-                    // convention in afw::geom::ellipses is to order moments (xx, yy, xy),
-                    // but the older algorithmic code uses (xx, xy, yy) - the order of
-                    // indices here is not a bug.
+                    // convention is to order moments (xx, yy, xy)
```

Comment:
Looks good, but can you smash the first two commits so there isn't a commit with the code change but not the doc change?

Reply:
Fair point